app/controllers/UserEventFire.php

```
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Support\Facades\Event;
+
+use Webhooks\Models\Subscription;
+
+use Webhooks\Models\Url;
+use Webhooks\Models\User;
+
+class UserEventFire extends \BaseController 
+{
+
+	public function webhookfire ($eventname,$payload)
+	{
+
+		$eveid = Webhooks\Models\Event::where('name',$eventname)->get();
+		@$eventid=$eveid[0]['id'];
+		
+		if (is_null ( @$eventid ))
```

check if empty instead of suppressing errors